### ISSUE TITLE
System Error when running Ganache 2.0.0-beta.2 on linux

### ISSUE THREAD

**Issue by aisuzuki** [trufflesuite/ganache#1106]
Error occurs when I add "Truffle project" using the new interface. 

Step to reproduce:
1. Launch the app
2. Click "NEW WORKSPACE"
3. Click "ADD PROJECT" and select arbitrary truffle.js file.
4. Click "SAVE WORKSPACE"
5. Error dialogue appears.

PLATFORM: linux
GANACHE VERSION: 2.0.0-beta.2

EXCEPTION:
```
Error: module.js:545
    throw err;
    ^

Error: Cannot find module '/src/truffle-project-loader/index.js'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.Module._load (module.js:470:25)
    at Function.Module.runMain (module.js:690:10)
    at startup (bootstrap_node.js:194:16)
    at bootstrap_node.js:666:3

    at Socket.child.stderr.on.data (/src/truffle-integration/projectDetails.js:31:15)
    at emitOne (events.js:115:13)
    at Socket.emit (events.js:210:7)
    at addChunk (_stream_readable.js:252:12)
    at readableAddChunk (_stream_readable.js:239:11)
    at Socket.Readable.push (_stream_readable.js:197:10)
    at Pipe.onread (net.js:589:20)
```

**Comment by honestbonsai** [comment]
Possibly fixed by #1149

**Comment by honestbonsai** [comment]
@aisuzuki Can you try this again? This might have been fixed by some recent PRs.

**Comment by aisuzuki** [comment]
The issue was fixed in 2.0.0-beta.2. Thank you very much!

**Comment by honestbonsai** [comment]
Closing issue since it's been resolved in 2.0.0-beta.2 as confirmed by @aisuzuki
